packages/components/addon/components/hds/segmented-group/index.hbs

```diff
@@ -0,0 +1,15 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<div class={{this.classNames}} ...attributes>
+  {{yield
+    (hash
+      Button=(component "hds/button" contextualClass="hds-button--segmented")
```

Comment:
Are we sure we want to do this? It's not compatible with embroider. (cc @Dhaulagiri since we talked about it earlier today)

Reply:
Can you say more about what aspect of this is not compatible? What is recommended as an alternative?